core/xml.js

```diff
@@ -622,7 +625,7 @@ const domToBlock = function(xmlBlock, workspace) {
       topBlockSvg.updateDisabled();
       // Allow the scrollbars to resize and move based on the new contents.
       // TODO(@picklesrus): #387. Remove when domToBlock avoids resizing.
-      workspace.resizeContents();
+      /** @type {!WorkspaceSvg} */ (workspace).resizeContents();
```

Comment:
Since it appears that only `SvgWorkspace`s can be `.rendered`, maybe add an `instanceof` check on line 607 instead of the cast here?